qaframework-bdd-tests/src/test/resources/features/refapp-2.x/07-visit/visits.feature

```diff
@@ -1 +1,46 @@
-Feature: Visits management
\ No newline at end of file
+Feature: Visits management
+
+  @selenium
+  @visit
+  Scenario: Starting a visit
+    Given user logins into the system
+    When user clicks on the find patient record app
+    And user searches for the patient record
+    And user clicks on the first patient record
+    And user clicks on the start visit link
+    Then the system starts the patient visit
+
+  @selenium
+  @visit
+  Scenario: Adding a past visit
+    Given user logins into the system
+    When user clicks on the find patient record app
+    And user searches for the patient record
+    And user clicks on the first patient record
+    And user clicks on the add past visit link
+    Then the system adds the patient past visit
+
+  @selenium
+  @visit
+  Scenario: Merging visits
+    Given user initiates login with two patient visits
```

Comment:
what does the step above accomplish?

Reply:
for `merging visits` test, i need a patient to be created `initially` with two visits. Hence i need to do that as shown in this code snippet.
```
    @Given("user initiates login with two patient visits")
    public void setUpLoginWithVisits() {
    	initiateLogin();
    	new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
    	new TestData.TestVisit(patient.uuid, TestData.getAVisitType(), getLocationUuid(homePage)).create();
    }
```